src/libs/error.mock.ts

```diff
@@ -9,13 +9,23 @@ import _ from 'lodash/fp';
  */
 export const errorWatcher = jest.fn();
 
+export const mockWithErrorReporting = _.curry((title, fn) => {
+  return async (...args) => {
+    try {
+      return await fn(...args);
+    } catch (error) {
+      errorWatcher(title, error);
+    }
+  };
+});
+
 /**
  * Provides a mocked version of error module's withErrorReportingInModal
  */
 export const mockWithErrorReportingInModal = _.curry((title, onDismiss, fn) => {
-  const errorHandler = async () => {
+  const errorHandler = async (...args) => {
```

Comment:
Discovered this bug after copying this function for `mockWithErrorReporting`. The real `withErrorReporting` functions pass arguments through to the wrapped function. This updates the mock to match the real behavior.